app/src/main/java/org/fossasia/openevent/general/similarEvents/SimilarEventsViewModel.kt

```diff
@@ -0,0 +1,55 @@
+package org.fossasia.openevent.general.similarEvents
+
+import android.arch.lifecycle.MutableLiveData
+import android.arch.lifecycle.ViewModel
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.schedulers.Schedulers
+import org.fossasia.openevent.general.event.Event
+import timber.log.Timber
+
+class SimilarEventsViewModel(private val eventTopicService: EventTopicService) : ViewModel() {
+
+    private val compositeDisposable = CompositeDisposable()
+
+    val progress = MutableLiveData<Boolean>()
+    val similarEvents = MutableLiveData<List<Event>>()
+    val error = MutableLiveData<String>()
+
+    fun loadSimilarEvents(id: Long) {
+        if(id.toString() == "-1"){
```

Comment:
why a conversion to string ? You can compare that directly

Reply:
It's not working if I use id.equals(-1).

Reply:
why dont you do id == -1

Reply:
id is of type long actually, so an error is cropping up

Reply:
Updated PR Please review :+1:  . Didn't change the string conversion

Reply:
Use id == -1